docs/src/api.md

```diff
@@ -199,19 +210,18 @@ Spaces.Quadratures.orthonormal_poly
 ```@docs
 Spaces.dss_transform
 Spaces.dss_untransform
-Spaces.dss_interior_faces!
 Spaces.dss_local_vertices!
-Spaces.dss_ghost_faces!
-Spaces.dss_ghost_vertices!
 Spaces.dss_local!
 Spaces.dss_local_ghost!
 Spaces.dss_ghost!
 Spaces.create_dss_buffer
 Spaces.DSSBuffer
+Spaces.create_ghost_buffer
 Spaces.weighted_dss_start!
 Spaces.weighted_dss_internal!
 Spaces.weighted_dss_ghost!
 Spaces.weighted_dss!
+Spaces.weighted_dss
```

Comment:
There is a 
![image](https://github.com/CliMA/ClimaCore.jl/assets/39932030/123e8406-084d-4c3c-a350-0db0c8fd4cdd)

Reply:
Yeah, this PR isn't fixing all of them. I'll try to close the rest out in a folloup PR, where we can make `strict = true`. I'll remove this since `weighted_dss` is not an existing function.